packages/snaps-cli/src/webpack/config.ts

```diff
@@ -265,10 +266,11 @@ export async function getDefaultConfiguration(
 
       /**
        * The `EnvironmentPlugin` is a Webpack plugin that adds environment
```

Comment:
We should update this comment and perhaps mention that the keys being defined are `process.env.X`